lady_luck.js

```diff
@@ -21,21 +18,14 @@ if (fs.existsSync('stored_results.json')) {
 
 var d10sold = d10s;
 
-function isAdmin(member) {
-	return member.hasPermission("ADMINISTRATOR");
-}
-
-function randInt(min, max) {
-	return min + Math.round(Math.random() * (max - min));
-}
+const isAdmin = member => member.hasPermission("ADMINISTRATOR")
+const randInt = (min,max) => min + Math.round(Math.random() * (max - min))
 
 function d10() {
-	ret = d10s.pop();
-	if (typeof(ret) == 'undefined') {
-		return randInt(1, 10);
-	} else {
-		return ret;
-	}
+	const ret = d10s.pop();
+	return typeof ret == 'undefined'
+	  ? randInt(1, 10)
+	  : ret
```

Comment:
This is less readable than a standard `if` statement. Also, you didn't terminate the line with a semicolon (`;`)